otelcollector/deploy/dependentcharts/prometheus-node-exporter/templates/_helpers.tpl

```diff
@@ -55,8 +55,8 @@ release: {{ .Release.Name }}
 Selector labels
 */}}
 {{- define "prometheus-node-exporter.selectorLabels" -}}
-app: {{ template "prometheus-node-exporter.name" . }}
-release: {{.Release.Name }}
+app.kubernetes.io/name: {{ include "prometheus-node-exporter.name" . }}
```

Comment:
this will break upgrades for node exporter in ARC, as selector labels are changing (from app to app.kubernetes.io) ?